Automattic/woocommerce-payments
# Fix how we detect if WCS is installed via WP CLI

Fixes 4385-gh-woocommerce/woocommerce-subscriptions

#### Changes proposed in this Pull Request

Fix how we detect if WooCommerce Subscriptions is activated via [wp-cli](https://developer.wordpress.org/cli/commands/plugin/activate/).

#### Testing instructions

* Make sure WooCommerce Payments installed and active
* Make sure WooCommerce Subscriptions are installed but inactive.
* Run `wp --path="{wordpress_path}" plugin activate woocommerce-subscriptions`.

With base branch, `woocommerce-subscriptions` will not be activated and a fatal error `CRITICAL Cannot declare class WCS_Core_Autoloader, because the name is already in use` will be logged. To check the fatal error, go to wp-admin > WooCommerce > Status > Logs > select most current fatal errors log.

With head branch, `woocommerce-subscriptions` will be successfully activated without fatal error.

-------------------


**Post merge**
